perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -117,15 +142,17 @@ def Start(vm):
   Args:
     vm: The target vm. Should already be configured via 'Configure'.
   """
-  vm.RemoteCommand(
-      'nohup {0}/bin/cassandra -p "{1}" 1> {2} 2> {3} &'.format(
-          CASSANDRA_DIR, CASSANDRA_PID, CASSANDRA_OUT, CASSANDRA_ERR))
+  if vm.OS_TYPE != JUJU:
```

Comment:
please reference os_types.JUJU directly rather than redefining a JUJU constant in this module